src/DynamoCoreWpf/ViewModels/Core/PortViewModel.cs

```diff
@@ -381,19 +251,6 @@ public SolidColorBrush PortBorderBrushColor
             }
         }
 
-        /// <summary>
-        /// Sets the color of the small rectangular marker on each input port.
-        /// </summary>
-        public SolidColorBrush PortValueMarkerColor
```

Comment:
What about obsoleting this public property or was it introduced in 2.13?

Reply:
I introduced this very recently